uncertainties/core.py

```diff
@@ -1772,489 +549,11 @@ def __str__(self):
-        ########################################
-
-        # Format of the main (i.e. with no exponent) parts (the None
-        # presentation type is similar to the g format type):
-
-        main_pres_type = 'fF'[(pres_type or 'g').isupper()]
-
-        # The precision of the main parts must be adjusted so as
-        # to take into account the special role of the decimal
-        # point:
-        if signif_limit is not None:  # If signif_limit is pertinent
-            # The decimal point location is always included in the
-            # printed digits (e.g., printing 3456 with only 2
-            # significant digits requires to print at least four
-            # digits, like in 3456 or 3500).
-            #
-            # The max() is important for example for
-            # 1234567.89123+/-12345.678 with the f format: in this
-            # case, signif_limit is +3 (2 significant digits necessary
-            # for the error, as per the PDG rules), but the (Python
-            # float formatting) precision to be used for the main
-            # parts is 0 (all digits must be shown).
-            #
-            # The 1 for the None pres_type represents "at least one
-            # digit past the decimal point" of Python
-            # (https://docs.python.org/3.4/library/string.html#format-specification-mini-language). This
-            # is only applied for null uncertainties.
-            prec = max(-signif_limit,
-                       1 if pres_type is None and not std_dev
-                       else 0)
-        ## print "PREC", prec
-
-        ########################################
-
-        # print (
-        #     "FORMAT_NUM parameters: nom_val_mantissa={},"
-        #     " std_dev_mantissa={}, common_exp={},"
-        #     " match.groupdict()={}, prec={}, main_pres_type={},"
-        #     " options={}".format(
-        #     nom_val_mantissa, std_dev_mantissa, common_exp,
-        #     match.groupdict(),
-        #     prec,
-        #     main_pres_type,
-        #     options))
-
-        # Final formatting:
-        return format_num(nom_val_mantissa, std_dev_mantissa, common_exp,
-                          match.groupdict(),
-                          prec=prec,
-                          main_pres_type=main_pres_type,
-                          options=options)
-
-    # Alternate name for __format__, for use with Python < 2.6 (and
-    # other Python versions if the user so chooses: this helps moving
-    # code from Python 2.6 to more recent versions):
-    @set_doc("""
-        Return the same result as self.__format__(format_spec), or
-        equivalently as the format(self, format_spec) of Python 2.6+.
-
-        This method is meant to be used for formatting numbers with
-        uncertainties in Python < 2.6, with '... %s ...' %
-        num.format('.2e').
-        """)
-    def format(*args, **kwargs):
-        return args[0].__format__(*args[1:], **kwargs)
-
+        return format_ufloat(self, format_spec)
```

Comment:
This lost a big docstring (now in format_ufloat). The `__format__` docstring is currently referenced by the `format_num` docstring.